code/lib/csf-tools/src/ConfigFile.ts

```diff
@@ -718,27 +716,11 @@ export const loadConfig = (code: string, fileName?: string) => {
 };
 
 export const formatConfig = (config: ConfigFile) => {
-  const { code } = generate.default(config._ast, {});
-  return code;
+  return printConfig(config).code;
 };
 
 export const printConfig = (config: ConfigFile, options: Options = {}) => {
-  const result = recast.print(config._ast, options);
-  const prettierConfig = prettier.resolveConfig.sync('.');
-
-  if (prettierConfig) {
-    let pretty: string;
-    try {
-      pretty = prettier.format(result.code, {
-        ...prettierConfig,
-        filepath: config.fileName ?? 'main.ts',
-      });
```

Comment:
@kasperpeulen can we detect the existence of `node_modules/.bin/prettier` and call it with the right arguments if it exists in a child_process?

That way we get the best of both worlds!?